app/services/VatPartialBuilder.scala

```diff
@@ -48,8 +48,14 @@ class VatPartialBuilderImpl @Inject() (val enrolmentsStore: EnrolmentsStoreServi
       case VatData(accountSummaryData, calendar) => accountSummaryData match {
```

Comment:
This file has some warnings (length exceeds 160, cyclomatic complexity) Not major but for me the full file is highlighted.